test/style/single_node_test.exs

```diff
@@ -12,7 +12,8 @@ defmodule Styler.Style.SingleNodeTest do
   use Styler.StyleCase, async: true
 
   test "charlist literals: rewrites single quote charlists to ~c" do
-    assert_style("'foo'", ~s|~c'foo'|)
+    assert_style("'foo'", ~s|~c"foo"|)
+    assert_style(~s|'"'|, ~s|~c"\\""|)
```

Comment:
`~s|'"'|` is one of the sillier elixir things i've ever personally written, glad you got to experience it too